enclave-tls/src/verifiers/api/enclave_verifier_register.c

```diff
@@ -51,7 +41,7 @@ enclave_verifier_err_t enclave_verifier_register(const enclave_verifier_opts_t *
 
 	/* Default type equals to name */
 	if (new_opts->type[0] == '\0')
-		strcpy(new_opts->type, new_opts->name);
+		strncpy(new_opts->type, new_opts->name, sizeof(new_opts->type));
```

Comment:
Plz fix this to:
```C
if (new_opts->type[0] == '\0') {
    strncpy(new_opts->type, new_opts->name, sizeof(new_opts->type) - 1);
    new_opts->type[sizeof(new_opts->type) - 1] = '\0';
}
```

Please apply this fix to all instances.

Reply:
all has modify snprintf